### ISSUE TITLE
How to complete remove DarkReader?

### ISSUE THREAD

**Issue by glennposadas** [darkreader/darkreader#2298]
Uninstalling seems won't return the default configs of Chrome.  Toggling everything on DarkReader does not work, so I would like to return things to normal.

**Comment by Gusted** [comment]
As I see you use Chrome so let's see where the problem persist here, because I've never had problems with uninstalling darkreader.

> Go to chrome://extensions/
> Search darkreader
> Press on Remove
> Check with a site of it's white again

**Comment by glennposadas** [comment]
Thanks, actually that didn't reset anything at all. I've tried that twice. I've tried removing cookings. I've tried resetting Chrome settings to default. I tried re-installing Chrome. Nothing worked! I was really frustrated coz it messed up all the websites I most of the time use (e.g. gmail, facebook, etc...).

BUT WHAT WORKED THOUGH was removing Chrome folder in Application Support > Google > Chrome. Voila. Thanks! 

This should help the others in the future. 
Note that this will reset everything, good thing there's a sync feature in Chrome to bring back your own settings.
